RedHatInsights/insights-advisor-frontend
# fix(Routing): Redirect to Recommendations tab by default

This fixes the problem when users navigate to `/insights/advisor` (the root level) and being redirected to `/insights/advisor/recommendations/pathways` instead of `/insights/advisor/recommendations`.

## How to test

1. Deploy with `npm run start:proxy:beta`,
2. Navigate to https://stage.foo.redhat.com:1337/preview/insights/advisor,
3. Check that you are redirected to https://stage.foo.redhat.com:1337/preview/insights/advisor/recommendations and the "Recommendations" tab is open,
4. Try the same for any non-existing page, the same redirect must be in place by default.
